### PR TITLE
feat: Add Security Assessments resource

### PR DESCRIPTION
**Pull request by irmatov**

### CODE REVIEW

File: resources/services/security/security_assessments_test.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
+package security
```
Comment (amanenk) [inline resources/services/security/security_assessments_test.go:1]: rename file to security_assessments_mock_test

File: resources/services/security/security_assessments.go [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+			},
+			{
+				Name:        "metadata_remediation_description",
+				Description: "Human readable description of what you should do to mitigate this security issue.",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("AssessmentProperties.Metadata.RemediationDescription"),
+			},
+			{
+				Name:     "metadata_categories",
+				Type:     schema.TypeStringArray,
```
Comment (amanenk) [inline resources/services/security/security_assessments.go:88]: Add description

**Review by amanenk — CHANGES_REQUESTED** [review]
a few small fixes required

File: resources/services/security/security_assessments_test.go (hunk 1 above)
Comment (irmatov) [reply resources/services/security/security_assessments_test.go:1]: Fixed, thanks.

File: resources/services/security/security_assessments.go (hunk 2 above)
Comment (irmatov) [reply resources/services/security/security_assessments.go:88]: Unfortunately there is no description in the SDK. :shrug:

**Review by disq — APPROVED** [review]

**Review by spangenberg — APPROVED** [review]

**Review by amanenk — APPROVED** [review]

**Pull request merged**